.github/workflows/backend.yml

```diff
@@ -42,10 +42,12 @@ jobs:
         with:
           python-version: '3.11'
       - name: Set up uv
-        uses: astral-sh/setup-uv@v2
+        uses: astral-sh/setup-uv@v4
         with:
           enable-cache: true
-          version: "0.4.13"
+          cache-dependency-glob: |
+            **/requirements/*.txt
```

Comment:
This is needed because the default in v4 is searching for `uv.lock`. Even the new default in the upcoming v5 won't work for our requirements.
https://github.com/astral-sh/setup-uv/issues/181